frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -88,6 +141,39 @@ pods:
           export KAFKA_OPTS="-Dsun.security.krb5.debug=true $KAFKA_OPTS"
         {{/TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
       {{/TASKCFG_ALL_SECURITY_KERBEROS_ENABLED}}
+      {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+          export SECURE_JMX_KEY_STORE_KEY=`cat $MESOS_SANDBOX/jmx/key_store_pass`
+          export KEY_STORE_PATH=$MESOS_SANDBOX/jmx/key_store
+          if [ -f "$MESOS_SANDBOX/jmx/trust_store" ]; then
+              # load user provided trust store
+              export TRUST_STORE_PATH=$MESOS_SANDBOX/jmx/trust_store
+              export SECURE_JMX_TRUST_STORE_KEY=`cat $MESOS_SANDBOX/jmx/trust_store_pass`
+          else
+              # load user provided keystore certs to default truststore
+              export TRUST_STORE_PATH=$JAVA_HOME/lib/security/cacerts
+              export SECURE_JMX_TRUST_STORE_KEY="changeit"
+              self_signed_aliases=$(${JAVA_HOME}/bin/keytool -list -v -keystore $KEY_STORE_PATH -storepass $SECURE_JMX_KEY_STORE_KEY | grep 'Alias name:' | awk '{print $3}')
+              for cert_alias in $self_signed_aliases ; do
+                  ${JAVA_HOME}/bin/keytool -importkeystore -srckeystore $KEY_STORE_PATH -destkeystore $TRUST_STORE_PATH -srcstorepass $SECURE_JMX_KEY_STORE_KEY -deststorepass $SECURE_JMX_TRUST_STORE_KEY -srcalias $cert_alias -destalias $cert_alias ;
+              done
+          fi
+
+          export JMX_PORT={{SECURE_JMX_PORT}}
+          export SECURE_JMX_RMI_PORT={{SECURE_JMX_RMI_PORT}}
+          export KAFKA_JMX_OPTS_SECRET="$KAFKA_JMX_OPTS_SECRET -Dcom.sun.management.config.file=$MESOS_SANDBOX/.ssl/jmx.properties"
+          export KAFKA_JMX_OPTS_SECRET="$KAFKA_JMX_OPTS_SECRET -Djava.rmi.server.hostname=$MESOS_CONTAINER_IP"
+
+          if [ ! -f "$MESOS_SANDBOX/jmx/key_file" ] ; then
+              echo "Secure JMX is enabled but no JMX pwd file was provided. Exiting."
```

Comment:
Since this is an error message I would be explicit and change `pwd` to `password`